internal/mode/static/nginx/config/servers_test.go

```diff
@@ -144,7 +144,7 @@ func TestExecuteServers(t *testing.T) {
 	}
 }
 
-func TestExecuteServers_IPFamily(t *testing.T) {
+func TestExecuteServerConfig(t *testing.T) {
```

Comment:
There's another test above `TestExecuteServers` that is basically doing the same type of tests. This test was originally targeted for just IP family. So either let's make a new test, or collapse tests.

Reply:
I vote for separate tests. It gets very hard to read the execute servers test when we are testing too many things